os/arch/arm/Kconfig

```diff
@@ -273,6 +273,7 @@ config ARCH_CORTEXA32
 	select ARCH_HAVE_RESET
 	select ARCH_HAVE_MMU
 	select ARCH_USE_MMU
+	select ARCH_HAVE_TESTSET
```

Comment:
Could you let me know why you add this?

Reply:
There are 2 reasons for adding this macro:
1) This is one of the dependency macro to enable CONFIG_SMP, thus it has to be enabled to have SMP appearing in the menuconfig selection
2) This will enable the spinlock mechanism to handle multicore coherency, you can view it from the file : os/arch/arm/src/armv7-a/arm_testset.S